### PR TITLE
Create governance.md

### PR DESCRIPTION
**Pull request by saad-ali**

### CODE REVIEW

**Comment by jdef** [comment]
is this ready for review? it looks like the "Accepting Changes" section was started (or, created) but there's no content

File: governance.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+The Container Storage Interace is an independently run open-source project (independent of any single cluster orchestration system, any single storage vendor, etc).
+It is proposed and maintained by members representing all major cluster orchestration (CO) systems (Kubernetes, Mesos, Docker, and Cloud Foundry).
+Contributions from the community are are welcome and accepted via pull requests.
+
+## Community Membership
+
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
```
Comment (jieyu) [inline governance.md]: nits. I'll align the separators `|`:
```
| Role | Responsibilities | Requirements | Defined by |
|------|------------------|--------------|------------|
```

File: governance.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+| member | active contributor in the community | sponsored by 2 reviewers.  multiple contributions to the project. | CSI GitHub org member. |
```
Comment (jieyu) [inline governance.md]: nits. Let's be consistent on Capitalization and period in the end.

File: governance.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+| reviewer | review contribution from other members | history of review and authorship | OWNERS file reviewer entry. |
+| approver | approve contributions for merge | core group of CO representatives | OWNERS file approver entry|
```
Comment (jieyu) [inline governance.md]: nits. looks like you leave a space before and after the separator. Let's make it consistent everywhere.

**Review by jieyu — APPROVED** [review]
lgtm. there are some style nits.

**Comment by saad-ali** [comment]
> lgtm. there are some style nits.

PTAL

> is this ready for review? it looks like the "Accepting Changes" section was started (or, created) but there's no content

Removed the "Accepting Changes" section. I think what's there stands pretty well on it's own. We can add to it as needed.

**Comment by julian-hj** [comment]
LGTM

**Comment by julian-hj** [comment]
@cpuguy83?

**Review by cpuguy83 — APPROVED** [review]
LGTM

File: governance.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+Contributions from the community are are welcome and accepted via pull requests.
+
+## Community Membership
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -----------|
+| member | Active contributor in the community. | Sponsored by 2 reviewers.  Multiple contributions to the project. | CSI GitHub org member. |
+| reviewer | Review contribution from other members. | History of review and authorship. | OWNERS file reviewer entry. |
+| approver | Approve contributions for merge. | Core group of CO representatives. | OWNERS file approver entry. |
+| janitor | Org and repo maintenance. | Minimal subset of approvers required for maintanence | GitHub repo write access. |
```
Comment (jdef) [inline governance.md]: nit: missing a period at the end of janitor requirements.

otherwise LGTM

**Review by jdef — APPROVED** [review]

**Review by julian-hj — APPROVED** [review]

**Pull request merged**